RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -201,13 +207,115 @@
                                     skipClusters = cms.InputTag('convClusters'),
                                     )
                                 )
-
+eras.trackingPhase1.toModify(convLayerPairs,
+    layerList = [
+        'BPix1+BPix2',
+
+        'BPix2+BPix3',
+        'BPix2+FPix1_pos',
+        'BPix2+FPix1_neg',
+        'BPix2+FPix2_pos',
+        'BPix2+FPix2_neg',
+
+        'FPix1_pos+FPix2_pos',
+        'FPix1_neg+FPix2_neg',
+
+        'BPix3+TIB1',
+        'BPix3+TIB2',
+
+        'TIB1+TID1_pos',
+        'TIB1+TID1_neg',
+        'TIB1+TID2_pos',
+        'TIB1+TID2_neg',
+        'TIB1+TIB2',
+        'TIB1+TIB3',
+
+        'TIB2+TID1_pos',
+        'TIB2+TID1_neg',
+        'TIB2+TID2_pos',
+        'TIB2+TID2_neg',
+        'TIB2+TIB3',
+        'TIB2+TIB4',
+
+        'TIB3+TIB4',
+        'TIB3+TOB1',
+        'TIB3+TID1_pos',
+        'TIB3+TID1_neg',
+
+        'TIB4+TOB1',
+        'TIB4+TOB2',
+
+        'TOB1+TOB2',
+        'TOB1+TOB3',
+        'TOB1+TEC1_pos',
+        'TOB1+TEC1_neg',
+
+        'TOB2+TOB3',
+        'TOB2+TOB4',
+        'TOB2+TEC1_pos',
+        'TOB2+TEC1_neg',
+
+        #NB: re-introduce these combinations when large displaced
+        #    tracks, reconstructed only in TOB will be available
+        #    For instance think at the OutIn Ecal Seeded tracks
+        #'TOB3+TOB4',
+        #'TOB3+TOB5',
+        #'TOB3+TEC1_pos',
+        #'TOB3+TEC1_neg',
+        #
+        #'TOB4+TOB5',
+        #'TOB4+TOB6',
+        #
+        #'TOB5+TOB6',
+
+        'TID1_pos+TID2_pos',
+        'TID2_pos+TID3_pos',
+        'TID3_pos+TEC1_pos',
+
+        'TID1_neg+TID2_neg',
+        'TID2_neg+TID3_neg',
+        'TID3_neg+TEC1_neg',
+
+        'TEC1_pos+TEC2_pos',
+        'TEC2_pos+TEC3_pos',
+        'TEC3_pos+TEC4_pos',
+        'TEC4_pos+TEC5_pos',
+        'TEC5_pos+TEC6_pos',
+        'TEC6_pos+TEC7_pos',
+        'TEC7_pos+TEC8_pos',
+
+        'TEC1_neg+TEC2_neg',
+        'TEC2_neg+TEC3_neg',
+        'TEC3_neg+TEC4_neg',
+        'TEC4_neg+TEC5_neg',
+        'TEC5_neg+TEC6_neg',
+        'TEC6_neg+TEC7_neg',
+        'TEC7_neg+TEC8_neg'
+        #other combinations could be added
+    ],
+    BPix = dict(TTRHBuilder = "TTRHBuilderWithoutAngle4PixelPairs"),
+    FPix = dict(TTRHBuilder = "TTRHBuilderWithoutAngle4PixelPairs"),
+    TIB1 = dict(clusterChargeCut = dict(refToPSet_ = "SiStripClusterChargeCutNone")),
```

Comment:
NOTE: since the value just repeats `dict(clusterChargeCut = dict(refToPSet_ = "SiStripClusterChargeCutNone"))` you could declare that as a variable and just use the variable for each assigned value.
```python
_clusterChargeCut = dict(clusterChargeCut = dict(refToPSet_ = "SiStripClusterChargeCutNone"))
eras.trackingPhase1.toModify(convLayerPairs,
...
                           TIB1 = _clusterChargeCut,
                           TIB2 = _clusterChargeCut,
                           ...
)
```